daml-assistant/exe/DA/Daml/Assistant.hs

```diff
@@ -190,15 +190,19 @@ handleCommand env@Env{..} logger command = do
 runCommand :: Env -> Command -> IO ()
 runCommand env@Env{..} = \case
     Builtin (Version VersionOptions{..}) -> do
+        let useCache =
+              UseCache
+                { cachePath = envCachePath
+                , damlPath = envDamlPath
+                , forceReload = vForceRefresh
+                }
         installedVersionsE <- tryAssistant $ getInstalledSdkVersions envDamlPath
-        availableVersionsE <- tryAssistant $ refreshAvailableSdkVersions envCachePath
+        snapshotVersionsEUnfiltered <- tryAssistant $ fst <$> getAvailableSdkSnapshotVersions useCache
```

Comment:
Is this just snapshot versions or all versions?

Reply:
Snapshot versions retrieves all possible versions, and then `getAvailableSdkVersions` extracts releases.

I'll change this naming a bit.